src/index.ts

```diff
@@ -1,13 +1,12 @@
 // ----- Types ----- //
 
-export type { Image } from './image';
-export { Role } from './image';
-export type { Lightbox } from './lightbox';
-export type { Sizes } from './sizes';
-
+export type { Image } from "./image";
+export { Role } from "./image";
+export type { Lightbox } from "./lightbox";
+export type { Sizes } from "./sizes";
 
 // ----- Components ----- //
 
-export { default as BodyImage } from './components/bodyImage';
-export { default as FigCaption } from './components/figCaption';
-export { default as Img } from './components/img';
```

Comment:
Is this going to work?

Reply:
In the same file further up, we're already doing `export { Role } from './image';` so `export { Img } from './components/img';` should work the same.